### PR TITLE
ffmpeg: update to 4.3.2.

### PR DESCRIPTION
**Pull request by jnbr**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/ffmpeg/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'ffmpeg'
 # audacity also needs to be bumped when a new ffmpeg version bumps libavformat's soname!
 pkgname=ffmpeg
-version=4.3.1
+version=4.3.2
 revision=7
```
Comment (biopsin) [inline srcpkgs/ffmpeg/template:5]: revision should be 1

Comment (jnbr) [reply srcpkgs/ffmpeg/template:5]: Good catch, thanks.

**Comment by ericonr** [comment]
Ping :)

**Comment by jnbr** [comment]
CI fails because dolphin-emu isn't cleaned from i686 repo.

**Pull request merged**